pgxn/neon/neon--1.0.sql

```diff
@@ -22,6 +22,16 @@ AS 'MODULE_PATHNAME', 'backpressure_throttling_time'
 LANGUAGE C STRICT
 PARALLEL UNSAFE;
 
+CREATE FUNCTION local_cache_hits() RETURNS bigint
+AS 'MODULE_PATHNAME', 'local_cache_hits'
+LANGUAGE C PARALLEL SAFE;
+
+
+CREATE FUNCTION local_cache_misses() RETURNS bigint
+AS 'MODULE_PATHNAME', 'local_cache_misses'
+LANGUAGE C PARALLEL SAFE;
+
+
```

Comment:
I wonder what is the best way to provide this (and may be some other information in future)?
You are absolutely right, I always forgot that  we have thousands of users of our neon extension and we really need to pay attention to any change in its API.

May be add some generic function, something like this:
```
create function get_neon_value(key text) returns text;
```
which can be used not only for local cache hiots/misses but also for any other characteristics/parameters/statistics we want to add in future.

Alternatively we can try to somehow push this information in pg_stat, although it seems to be more difficult.

Reply:
>I wonder what is the best way to provide this (and may be some other information in future)?

0) We need to actually run `create extension neon` and/or `ALTER EXTENSION neon UPDATE` for all computes.
I've created issue to discuss this: https://github.com/neondatabase/neon/issues/5726

1) I think keeping this stuff in `neon` extension is a good idea, because in extension we can iterate without touching postgres code (which is imho, great).

2)  "-1" from me to generic functions
Let's keep API clear and call functions in a meaningful way.
There's no harm in extra `CREATE OR UPDATE function`.

3) To fix forgetfulness, I suggest to add some comment on top of the neon--*.sql file. Something like:
```
Do not fix existing extension scripts, add new script, change default version in neon.control.
Also update `compute_ctl` "ALTER EXTENSION neon UPDATE TO '1.1'
```
Or even better - add automated CI check, that these files are not touched and all version numbers are updated.

Reply:
> 0. We need to actually run create extension neon 

+1
May be you should create PR for it?

> 2. "-1" from me to generic functions

Well, I do not quite understand why `get_local_file_cache_hits()` function is 
better than `neon_stats` view.  It is hard for user to remember names of all this admin functions, for example I even after ~10 years of active Postgres development has to ask google how to get current xid:)
And in the view you can see all available keys and find one that you need.